abstract-iterator.js

```diff
@@ -470,6 +413,34 @@ for (const k of ['_ended property', '_nexting property', '_end method']) {
   })
 }
 
+function assertStatus (iterator) {
+  if (iterator[kClosing]) {
+    throw new ModuleError('Iterator is not open: cannot read after close()', {
+      code: 'LEVEL_ITERATOR_NOT_OPEN'
+    })
+  } else if (iterator[kWorking]) {
+    throw new ModuleError('Iterator is busy: cannot read until previous read has completed', {
+      code: 'LEVEL_ITERATOR_BUSY'
+    })
+  } else if (iterator[kAbortController].signal.aborted) {
+    throw new AbortedError()
```

Comment:
Comment this out and add a TODO comment. We might not need it (if aborting will close the iterator, TBD) and removing this behavior will be harder than adding it.